package.json

```diff
@@ -1,42 +1,62 @@
 {
   "name": "taskcluster-client",
-  "version": "1.2.1",
+  "version": "2.0.0",
   "author": "Jonas Finnemann Jensen <jopsen@gmail.com>",
   "description": "Client for interfacing taskcluster components",
   "license": "MPL-2.0",
+  "files": [
+    "build",
+    "apis.json"
+  ],
+  "main": "build/index.js",
+  "browser": "build/browser.js",
   "scripts": {
-    "test": "./test/runtests.sh"
-  },
-  "browser": "./browser.js",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/taskcluster/taskcluster-client.git"
+    "build:browser": "webpack",
+    "build:node": "babel src --out-dir build --source-maps inline --ignore node_modules",
+    "build": "npm run build:node && npm run build:browser",
```

Comment:
Also these builds script seems like a lot of typing.. we can avoid them all using `babel-compile`, maybe it could have a better name, but it's consistent and we don't have type and copy/paste stuff around between repos... and no need to do `.babelrc` and depend on other babel plugins..